main.go

```diff
@@ -166,3 +166,11 @@ func runControllers(ctx context.Context, kubeClient kubernetes.Interface, cfg *r
 
 	nodeIpamController.Run(ctx)
 }
+
+func bindingAddress(cfg config) string {
+	if cfg.HealthProbeAddr != "" {
```

Comment:
By default `HealthProbeAddr` is empty now. Add this check for existing clusters.
The default addr value `:8081` from `HealthProbeAddr` is now specified for `WebserverBindAddr`